src/test_workflow/test_args.py

```diff
@@ -33,6 +34,7 @@ def __init__(self) -> None:
         parser.add_argument("--test-run-id", type=int, help="The unique execution id for the test")
         parser.add_argument("--component", type=str, dest="components", nargs='*', help="Test a specific component or components instead of the entire distribution.")
         parser.add_argument("--keep", dest="keep", action="store_true", help="Do not delete the working temporary directory.")
+        parser.add_argument("--ft-repo-ref", type=str, default=None, dest="ft_repo_ref", help="Specify GitHub ref for functionalTestDashboards repo to override default one from build manifest.")
```

Comment:
Suggest `--functional-test-repo-ref`, or anything that is consistent across the board just like mentioned above.